jfr-connection/src/main/java/io/opentelemetry/contrib/jfr/connection/FlightRecorderDiagnosticCommandConnection.java

```diff
@@ -197,41 +224,41 @@ public void closeRecording(long id) {
         "closeRecording not available through the DiagnosticCommand connection");
   }
 
-  // Do this check separate from assertCommercialFeatures because reliance
-  // on System properties makes it difficult to test.
-  static boolean jdkHasUnlockCommercialFeatures(MBeanServerConnection mBeanServerConnection) {
-    try {
-      RuntimeMXBean runtimeMxBean =
-          ManagementFactory.getPlatformMXBean(mBeanServerConnection, RuntimeMXBean.class);
-      String javaVmVendor = runtimeMxBean.getVmVendor();
-      String javaVersion = runtimeMxBean.getVmVersion();
-      return javaVmVendor.contains("Oracle Corporation")
-          && javaVersion.matches("(?:^1\\.8|9|10).*");
-    } catch (IOException e) {
-      return false;
-    }
-  }
-
   // visible for testing
   static void assertCommercialFeaturesUnlocked(
```

Comment:
Should it be renamed now to `unlockCommercialFeatures`?

Reply:
This [Readme note](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/26c31db412405271e200fa876431f8e13d5db0b4/jfr-connection/README.md?plain=1#L33) should perhaps be removed now.

Reply:
I think the readme note should stay since it does provide information about JFR and certain versions of Java.